hoot-services/src/main/java/hoot/services/controllers/ogr/ExtractAttributesResource.java

```diff
@@ -191,7 +191,10 @@ public Response getAttributes(@PathParam("id") String jobId,
                         files.addAll(processFolder(folderWithUnzippedContents));
                     }
                     else {
-                        files.add(uploadedFile);
+                        //if we have shapefiles, only include shp refs
+                        if (extension.equalsIgnoreCase("shp")) {
+                            files.add(uploadedFile);
+                        }
```

Comment:
I don't know if OGR behavior changed after a GDAL upgrade, but the extraneous components of shapefile as inputs were causing a `A valid reader could not be found for the URL` error, specifically for the `shx` file.

Reply:
This file list ends up being passed to `hoot tag-info` command

Reply:
That's weird.  Might have happened when I was refactoring all the convert command logic into a single class awhile back.  I remember that broke something else that Matt had previously worked on.